### PR TITLE
Chapter 7 : Sign up

### PR DESCRIPTION
**Pull request by VanNguyen21**
screenshot from 2019-02-19 11-14-12

### CODE REVIEW

**Comment by VanNguyen21** [comment]
Ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -4,4 +4,21 @@ def show
...
 
   def new; end
+
+  def create
+    @user = User.new(params[:user])
+    if @user.save
+      flash[:success] = I18n.t "welcome"
+      redirect_to @user
+    else
+      render "new"
```
Comment (phamvanbk11) [inline app/controllers/users_controller.rb]: :new

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -4,4 +4,21 @@ def show
   end
 
   def new; end
+
+  def create
+    @user = User.new(params[:user])
```
Comment (phamvanbk11) [inline app/controllers/users_controller.rb]: xóa ()

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -4,4 +4,21 @@ def show
...
+    if @user.save
+      flash[:success] = I18n.t "welcome"
```
Comment (phamvanbk11) [inline app/controllers/users_controller.rb]: t "welcome" là đc rồi

**Comment by phamvanbk11** [comment]
commented

**Comment by VanNguyen21** [comment]
Ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**